src/App.js

```diff
@@ -1,24 +1,28 @@
-import logo from './logo.svg';
 import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import AllBeers from './components/AllBeers';
+import NewBeer from './components/NewBeer';
+import RandomBeers from './components/RandomBeers';
+import HomePage from './components/HomePage';
+import NavBar from './components/NavBar';
+import SingleBeer from './components/SingleBeer';
+
+
+
+
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <div>
+    <NavBar/>
```

Comment:
Since you already call the `<NavBar/>` inside the other components, you don't need to call it here. This is why, when you go to `/all-beers` you have 2 navbars.